lib/hexdocs/queue.ex

```diff
@@ -1,194 +1,135 @@
 defmodule Hexdocs.Queue do
-  def name() do
-    Application.get_env(:hexdocs, :queue_name)
+  use Broadway
+  require Logger
+
+  @ignore_packages ~w(eex elixir ex_unit iex logger mix hex)
+
+  def start_link(_opts) do
+    name = Application.fetch_env!(:hexdocs, :queue_name)
+    producer = Application.fetch_env!(:hexdocs, :queue_producer)
+
+    Broadway.start_link(__MODULE__,
+      name: __MODULE__,
+      producers: [
+        default: [
+          module: {producer, queue_name: name, max_number_of_messages: 10, wait_time_seconds: 10},
+          stages: 1
+        ]
+      ],
+      batchers: [
+        default: []
+      ],
+      processors: [
+        default: [stages: 1]
+      ]
+    )
```

Comment:
What kind of concurrency do we get out of this? Before we had 4 consumers.

Reply:
Oops, tgat was supposed to be 4 as well, I switched it when debugging and forgot to switch back. Good catch!